driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -275,9 +275,9 @@ public static Bson lookup(final String from, final List<? extends Bson> pipeline
      * @mongodb.server.release 3.6
      * @since 3.7
      */
-    public static Bson lookup(final String from, @Nullable final List<Variable<?>> let, final List<? extends Bson> pipeline,
-                              final String as) {
-       return new LookupStage(from, let, pipeline, as);
+    public static <TExpression> Bson lookup(final String from, @Nullable final List<Variable<TExpression>> let,
```

Comment:
This isn't binary breaking because of type erasure, so I don't think it requires a major release.

Anyone trying to call this method would have been stymied anyway, as you can only pass it null.